test/tests/haskell-stack/container.sh

```diff
@@ -1,7 +1,17 @@
 #!/bin/bash
-set -e
 
 # stack mostly sends to stderr
-stack --resolver ghc-$(ghc --print-project-version) new myproject 2> /dev/null
-cd myproject
-stack run 2> /dev/null
+RESULT=$(stack --resolver ghc-$(ghc --print-project-version) new myproject 2>&1 > /dev/null)
+
+# we only want to halt on error after this point
+set -e
```

Comment:
Ignoring errors is definitely not going to be something we'll accept :grimacing:

Here's the approach I'd suggest instead:

```bash
#!/bin/bash
set -e

# stack mostly sends to stderr
if ! stackResult="$(stack --resolver ghc-$(ghc --print-project-version) new myproject 2>&1 > /dev/null)"; then
	case "$stackResult" in
		*Unable to load global hints for*)
			echo >&2 'skipping; stack does not yet support this Haskell version'
			exit 0
			;;
		*)
			echo >&2 'error: stack failed:'
			echo >&2 "$stackResult"
			exit 1
			;;
	esac
fi

cd myproject
stack run 2> /dev/null
```

Reply:
Oh nice, I was trying to do something like this but failing :laughing: 

I have updated to something like this thanks!